queries.sql

```diff
@@ -158,3 +158,70 @@ GROUP BY full_name
 ORDER BY c DESC
 FETCH FIRST 1 ROWS WITH TIES;
 
+
+
+-- How many different animals did Stephanie Mendez see?
+
+SELECT Count(*) FROM animals
```

Comment:
- [ ] There is currently no query that answers the question `Who was the last animal seen by William Tatcher?`. Kindly add it.